rcongui/src/components/PlayerInfo/PlayerInfo.js

```diff
@@ -358,15 +368,23 @@ const PlayerInfoFunc = ({ classes }) => {
                 </Grid>
               </Grid>
             </Grid>
-            <Grid item xl={3} lg={3} md={3} sm={4} xs={12}>
+            <Grid item xl={3} lg={3} md={3} sm={4} xs={12} container>
```

Comment:
item AND container, that's not correct, it's either or. If you need a container you should remove the sizing parameters also